modules/serverless-ossm-secret-filtering-net-kourier.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * /serverless/knative-serving/config-custom-domains/domain-mapping-custom-tls-cert.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-ossm-secret-filtering-net-kourier_{context}"]
+= Improving net-kourier memory usage by using secret filtering for {SMProductShortName}
```

Comment:
I'd drop `for {SMProductShortName}`  , as it is unrelated to ServiceMesh

Reply:
@maschmid Forgot about this when moving the section; fixed now.